src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/feature_value.yml

```diff
@@ -0,0 +1,25 @@
+admin_feature_values_index:
+  path: /
+  methods: [ GET ]
+  defaults:
+    _controller: PrestaShopBundle\Controller\Admin\Sell\Catalog\FeatureValueController::indexAction
+    _legacy_controller: AdminFeatures
+    _legacy_link: AdminFeatures
```

Comment:
I think the legacy controller also depended on a feature_id parameter so you can also add:
```suggestion
    _legacy_link: AdminFeatures
    _legacy_parameters:
      id_feature: featureId
```